react/Select/Readme.md

```diff
@@ -0,0 +1,57 @@
+A combo box for more advanced use cases, based on [`react-select` v2](https://deploy-preview-2289--react-select.netlify.com/home).
+
+### Simple use case
+
+You can use this exactly like you would use `react-select`. See the [official documentation](https://deploy-preview-2289--react-select.netlify.com/home) for more examples.
+
+```
+const options = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry' },
+  { value: 'vanilla', label: 'Vanilla' }
+];
+
+<Select options={options} />
+```
+
+### Custom control component
+
+You can use the `withCustomControl` HOC to turn an existing, custom component into the control button for the `Select`:
+
+```
+const withCustomControl = require('../Select').withCustomControl;
```

Comment:
Do you think withCustomControl could be renamed reactSelectControl ? IMHO it would be clearer.